OpenRVS/classes/OpenLogger.uc

```diff
@@ -0,0 +1,16 @@
+class OpenLogger;
```

Comment:
Here is the new logger class. First time writing UnrealScript!

Reply:
Every unrealscript class has to extend a parent.  Object is the base parent of all classes in the hierarchy, and you don't need any specialized features of other classes, so in this case I'd recommend extending Object here.  eg
class OpenLogger extends Object;

Reply:
If no config file is specified, it will default to saving the variable in ravenshield.ini.  This is fine if that's what you like but we already have openrvs.ini for config variables so I would recommend putting this in there.  You'd do that this way:
class OpenLogger extends Object config(openrvs);